### PR TITLE
fix md prefix in the html file

### PR DESCRIPTION
**Pull request by ZhaoqiongZ**
fix md prefix in the html file
there are some .md link in the generated html file, replace them to .html before publish

### CODE REVIEW

**Review by yinghu5 — APPROVED** [review]
another way to fix the link broken issue, convert readme.md link in html table, verified in local machine 
http://clx03-tce.sh.intel.com:8080/_build/html/GenAIComps/comps/embeddings/README.html  
vs. broken link: https://opea-project.github.io/latest/GenAIComps/comps/embeddings/

**Review by dbkinder — CHANGES_REQUESTED** [review]
No. Please DO NOT approve and merge this PR.

Changing every .md to .html in the generated HTML files will break things.  For example, there's a link at the top of every generated page to the source file for that HTML page, something like `README.md.txt` that this script would change to an incorrect `README.html.txt`.   There are also quite valid mentions of .md files that need to stay as .md, such as a directory structure description that would be incorrect if the .md changed to .html:

```
community/CONTRIBUTING.html:│   │   │       ├── README.md
community/CONTRIBUTING.html:│   │           ├── how_to_validate_service.md    #optional
community/CONTRIBUTING.html:│   │           ├── README.md
community/CONTRIBUTING.html:│           ├── README.md
community/CONTRIBUTING.html:│   ├── README_gmc.md  # K8s quickstar
community/CONTRIBUTING.html:│   └── README.md      # quickstart
community/CONTRIBUTING.html:├── README.md
```

Another is in the community/rfcs.html that has an example of the naming convention for RFCs saying:

```
For example, 24-04-29-GenAIExamples-001-Using_MicroService_to_implement_ChatQnA.md
```
This PR would also break valid references to .md files in external references, such as `href="https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/workloads/opea/chatqna/README.md"`

And there are more cases like this.  The common reason we still have incorrect .md references in the generated HTML is because of raw HTML tags in markdown files that are not processed by Sphinx and just placed as-written into the generated HTML (that's what raw HTML is supposed to do).  We need to hunt down and remove unnecessary uses of raw HTML in markdown files, and in particular, raw HTML that includes links that Sphinx needs to see and convert as needed, for example, change raw HTML tables into normal markdown tables with markdown link syntax.

File: scripts/correct_prefix_to_html.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Specify the file path
+file_path = '_build/html/GenAIComps/README.html'
+
+# Read the file content
+with open(file_path, 'r', encoding='utf-8') as file:
+    content = file.read()
+
+# Replace .md with .html
+content = content.replace('.md', '.html')
```
Comment (dbkinder) [inline scripts/correct_prefix_to_html.py:9]: No. You must not change all .md to .html

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by dbkinder** [comment]
This PR is unnecessary would break the github.io site

**Pull request closed without merging**